plugins/chain_plugin/tracked_votes.cpp

```diff
@@ -34,10 +34,15 @@ namespace eosio::chain_apis {
             if (!_update_vote_block_metrics && !tracking_enabled && !chain::vote_logger.is_enabled(fc::log_level::info))
                return;
 
+            // do not bother tracking/logging when syncing or replaying
+            auto now = fc::time_point::now();
+            if (now - block->timestamp > fc::minutes(5) && (block->block_num() % 1000 != 0))
```

Comment:
Do we still want to do it every thousand blocks?

Reply:
I think so, gives a bit of status as it syncs.